### PR TITLE
Add usb page

### PR DESCRIPTION
**Pull request by sashcy**

### CODE REVIEW

File: src/templates/landing_pages/usb.jsx [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+            </div>
+        </div>
+        <div className='section-md primary-bg-color coin-section'>
+            <div className='container center-text'>
+                <div className='gr-row align-items-center'>
+                    <div className='gr-2 gr-12-m gr-12-p'>
+                        <img className='usb-coin' src={it.url_for('images/usb_page/usb-coin-logo.png')} alt='USB' />
+                    </div>
+                    <div className='gr-10 gr-12-m gr-12-p'>
+                        <p className='content-inverse-color ft-400'>{it.L('Guaranteed by [_1] USB is a digital currency pegged to the US Dollar on a one-to-one basis. Send and receive payments with the peace of mind that your currency is backed by the strength of [_1] balance sheet.', it.broker_name)}</p>
```
Comment (kellybinary) [inline src/templates/landing_pages/usb.jsx]: missing a comma after the sentence Guaranteed by Binary.com?: `Guaranteed by [_1], USB is a ...`

File: src/sass/landing_pages/usb.scss [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+    max-width: 150%;
+    height: 150px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+}
+
+.icon-buy {
+    background: url('../images/usb_page/buy.svg') no-repeat;
+    background-position: center;
```
Comment (kellybinary) [inline src/sass/landing_pages/usb.scss]: how about using only `background-image: url('..');` for the icons? `background-repeat` and `background-position` property has been declared in line 200 and 201 above

File: src/sass/landing_pages/usb.scss [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+}
+
+/* Video Embed */
+
+.video-container {
+    width: 100%;
+    height: 0px;
+    position: relative;
+    padding-bottom: 56.19%;
+    //box-shadow: 0 3px 28px 0 rgba(0, 0, 0, 0.03), 0 3px 28px 0 rgba(0, 0, 0, 0.3);
```
Comment (kellybinary) [inline src/sass/landing_pages/usb.scss]: uncomment or remove this line? :)

File: src/sass/landing_pages/usb.scss [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+
+.bottom-50 {
+    margin-bottom: 50px;
+}
+
+.dark-grey-bg {
+    background: #eaeaea;
+}
+
+.justify-content-center {
```
Comment (kellybinary) [inline src/sass/landing_pages/usb.scss]: i think there's existing class that is equivalent to this: `.gr-row-align-center`, you can use this instead

File: src/sass/landing_pages/usb.scss [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+    justify-content: center;
+}
+
+.align-items-center {
```
Comment (kellybinary) [inline src/sass/landing_pages/usb.scss]: here as well: `.gr-row-align-middle`

File: src/sass/landing_pages/usb.scss [hunk 6]
```diff
@@ -0,0 +1,423 @@
...
+    align-items: center;
+}
+
+.justify-content-between {
```
Comment (kellybinary) [inline src/sass/landing_pages/usb.scss]: and the equivalent class for this one would `.gr-row-align-between` :P

File: scripts/config/pages.js [hunk 7]
```diff
@@ -118,5 +118,6 @@ module.exports = [
     // ==================== Section: "landing_pages" ====================
     // ['graduates',                               'landing_pages/graduate_program',              null,        'Binary.com Graduate Program',        'NOT-en'],
     // ['hackathon',                               'landing_pages/hackathon',                     null,        'Hackathon Competition',              'NOT-en'],
-    ['landing/signup-frame',                    'landing_pages/signup_frame',                  null,        'Sign up'],
+    ['introducing-usb',                            'landing_pages/usb',                           null,        'Introducing USB'],
```
Comment (negarn) [inline scripts/config/pages.js]: if we are only releasing this for english, we should add the 'NOT-en' again to the end

File: src/templates/landing_pages/usb.jsx [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+                            <div className='mobile-hide binary-logo-text'>
+                                <div />
+                            </div>
+                        </a>
+                    </div>
+                </div>
+                <div className='nav-col-right'>
+                    <div id='push' className='grill grill-ico' />
+                    <div id='push-nav' className='topnav'>
+                        <a className='page-scroll' href='#video'>{it.L('Overview')}</a>
```
Comment (negarn) [inline src/templates/landing_pages/usb.jsx]: please remove all these `it.L`'s again since we don't want to translate this page 😞

**Pull request merged**